docs/configs/static/typescript.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
```

Comment:
This looks like we're copying files from other people here. Is this ok in terms of copyrights?

Reply:
It's what  get generated if you run ./fmt.sh script. Not sure who to ask regarding copyright. @bame-da any thoughts?